src/core_atmosphere/physics/mpas_atmphys_driver_lsm_noahmp.F

```diff
@@ -377,8 +377,10 @@ subroutine lsm_noahmp_fromMPAS(configs,mesh,diag,diag_physics,diag_physics_noahm
 !    see lines 242-290 in module NoahmpIOVarType.F90):
  call mpas_pool_get_array(output_noahmp,'t2mvxy'  ,t2mvxy  )
  call mpas_pool_get_array(output_noahmp,'t2mbxy'  ,t2mbxy  )
+ call mpas_pool_get_array(output_noahmp,'t2mxy'   ,t2mxy   )
```

Comment:
Is there a reason for retrieving a pointer to `t2mxy` but not using it in this routine? And similarly for `q2mxy`?